src/features/peers/redux/reducer.test.ts

```diff
@@ -0,0 +1,26 @@
+import { peers } from './reducer'
+
+const initialState: string[] = []
```

Comment:
Q: why initialState is this?

Reply:
initialState is an initially an empty array of type string[ ]

I don't know if i quite understand your question